web/src/index.html

```diff
@@ -36,5 +34,6 @@
       gtag('js', new Date());
       gtag('config', 'UA-120133131-1');
     </script>
+    <!-- built scripts will get loaded below here -->
   </body>
 </html>
```

Comment:
This was just being straight-up copied as part of the npm `build` script.  Now it's being used as a template by the webpack config, and the CSS and Javascript files are dynamically injected.  🎉